src/test/java/jalau/at18/katas/multiple/daniela/MainTest.java

```diff
@@ -0,0 +1,15 @@
+package jalau.at18.katas.multiples.daniela;
+
+import static org.junit.Assert.*;
+import org.junit.Test;
+
+public class MainTest {
+
+    @Test
+    public void functionShouldGiveHighest() {
```

Comment:
Not sure if test name is ok because this test is actually verifying method that sums multiples instead for give "highest"